Python-packages/covidcast-py/covidcast/covidcast.py

```diff
@@ -379,15 +368,15 @@ def _detect_metadata(data: pd.DataFrame,
     return unique_data_source_vals[0], unique_signal_col_vals[0], unique_geo_type_vals[0]
 
 
-def _fetch_single_geo(data_source: str,
-                      signal: str,  # pylint: disable=W0621
-                      start_day: date,
-                      end_day: date,
-                      geo_type: str,
-                      geo_value: str,
-                      as_of: date,
-                      issues: Union[date, tuple, list],
-                      lag: int) -> Union[pd.DataFrame, None]:
+def _fetch_epidata(data_source: str,
+                   signal: str,  # pylint: disable=W0621
+                   start_day: date,
+                   end_day: date,
+                   geo_type: str,
+                   geo_value: str,
+                   as_of: date,
```

Comment:
type annotation false: can't this now be a str or list of strs?

Reply:
ah yes, added